ui/components/show/showCards/ShowCards.tsx

```diff
@@ -12,72 +11,74 @@ import CardImgContainer from './ImgContainer/CardImgContainer';
 import TagsContainer from '../../layout/cardElements/TagsContainer/TagsContainer';
 import ContentSection from '../../layout/ContentSection/contentSection';
 
-import styles from "./ShowCards.module.scss";
+import { Swiper, SwiperSlide } from 'swiper/react';
+import 'swiper/swiper-bundle.min.css';
 
-interface ShowCardsProps{
-    shows: Show[]
-  }
+import styles from "./ShowCards.module.scss";
 
+interface ShowCardsProps {
+  shows: Show[];
+  singleRow?: boolean;  // New prop to control single row display
```

Comment:
This prop will not be new forever. If you would like a comment here remove 'New prop'. We should make it more descriptive of what this flag changes within the component.

```suggestion
  singleRow?: boolean;  // Should the collection of cards be displayed on a single horizontally scrolling row? 
```